BrainPortal/app/controllers/portal_controller.rb

```diff
@@ -40,7 +40,7 @@ def welcome #:nodoc:
     end
 
     @num_files              = current_user.userfiles.count
-    @groups                 = current_user.has_role?(:admin_user) ? current_user.groups.order(:name) : current_user.available_groups.order(:name)
+    @groups                 = current_user.has_role?(:admin_user) ? current_user.groups.order(:name) : current_user.viewable_groups.order(:name)
```

Comment:
With a NormalUser or a SiteManager I see `Everyone` group in the welcome page of CBRAIN.

Reply:
Will fix with an explicit exclusion